otter/auth.py

```diff
@@ -284,6 +284,42 @@ def __hash__(self):
                      self._url, self._admin_url))
 
 
+@implementer(IAuthenticator)
+class SingleTenantAuthenticator(object):
+    """
+    An authentication handler that authenticates using a single tenant id.
+    """
+    def __init__(self, _identity_user, _identity_password, url):
```

Comment:
If the user screws this up, wouldn't they just get authentication errors anyway?

(Also, more importantly, if someone has a custom identity endpoint that lets certain creds authenticate for any tenant, this change would prevent them from using that for ostensibly little to no benefit.)